src/app/(registration)/layout.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import CMSImage from "@/components/SharedComponents/CMSImage";
+import withFeatureFlag from "@/hocs/withFeatureFlag";
+import { getIllustrationAsset } from "@/image-data";
+import { OakBox, OakFlex, OakMaxWidth } from "@/styles/oakThemeApp";
+
+function RegistrationLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <OakBox $background={["white", "bg-decorative1-main"]} $overflow="hidden">
+      <OakMaxWidth
+        $justifyContent="center"
+        $alignItems={["center", "initial"]}
+        $height="100vh"
+      >
+        <CMSImage
+          image={getIllustrationAsset("auth0-background")}
```

Comment:
This image is really grainy. Pre-empting a future ticket to improve it, could we export it at 2x resolution?

Reply:
re-exported as svg 👁️ 👁️ 
<img width="1017" alt="Screenshot 2024-08-19 at 12 24 24" src="https://github.com/user-attachments/assets/508fef37-a47f-4dbf-ba45-d913aaafa2a0">